app/helpers/hosts_helper.rb

```diff
@@ -247,7 +248,12 @@ def show_templates
   end
 
   def overview_fields(host)
+    global_status = host.build_global_status
     fields = [
+      [_("Status"), content_tag(:i, '&nbsp'.html_safe, :class => host_global_status_icon_class(global_status.status)) +
+                    content_tag(:span, _(global_status.to_label), :class => host_global_status_class(global_status.status), :id => 'global_status_handler') +
+                    host_detailed_status_list(host, :class => 'hidden')
```

Comment:
Should this be expanded by default?  I think unless we have hundreds of status types that showing them would be beneficial.